testsuite/HOWTO.md

```diff
@@ -0,0 +1,120 @@
+The unit tests and the test data are bundled together in the package **MDAnalyisTests**. In order to run the tests, this package must be installed in addition to MDAnalysis.
+
+The tests also rely on the `pytest` and `numpy` packages, and require both to run.
+
+# Users
+Install [MDAnalysisTests](MDAnalysisTests) via **pip**
+```
+pip install --upgrade MDAnalysisTests
+```
+
+or via **conda**
+```
+conda config --add channels conda-forge
+conda install MDAnalysisTests
+```
+or download the tar file, unpack, and run `python setup.py install`.
+
+Run the tests by invoking
+```
+python -c 'from MDAnalysisTests import run; run()'
+```
+
+All tests should pass (i.e. no **FAIL**, **ERROR**); *SKIPPED* or *XFAIL* are ok. For anything that fails or gives an error [ask on the user mailing list](http://users.mdanalysis.org) or [raise an issue](/MDAnalysis/mdanalysis/issues).
+
+# Developers #
+
+We are borrowing some of NumPy's testing frame work; thus, numpy **must** be installed for the tests to run at all.
+
+
+It is recommended that you run the tests from the [git source repository](Source), which are located in the [testsuite/MDAnalysisTests](https://github.com/MDAnalysis/mdanalysis/tree/develop/testsuite) directory:
+```
+cd testsuite/MDAnalysisTests
+pytest  --disable-pytest-warnings
```

Comment:
please explain why this variable is set. I would also add a recommendation to not use the flag when single files/tests are run to see what if any warnings the code is throwing.